src/StackAdmin/Storage.Management/Commands.Management.Storage/packages.config

```diff
@@ -4,6 +4,6 @@
   <package id="Microsoft.Data.Edm" version="5.6.4" targetFramework="net40" />
   <package id="Microsoft.Data.OData" version="5.6.4" targetFramework="net40" />
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net40" />
-  <package id="Microsoft.WindowsAzure.Management.Storage" version="5.1.1" targetFramework="net40" />
+  <package id="Microsoft.WindowsAzure.Management.Storage" version="6.0.1" targetFramework="net40" />
```

Comment:
is this intentional?

Reply:
Nope. Changed it back and fixed a different packages.config issue.